install_config/monitoring/modules/applying-custom-alertmanager-configuration.adoc

```diff
@@ -0,0 +1,60 @@
+= Applying custom Alertmanager configuration
+
+The default configuration of the {product-title} Monitoring Alertmanager cluster is:
```

Comment:
Please move this example into the procedure body or create a separate reference module.